gci/componentmodel.py

```diff
@@ -654,7 +651,7 @@ def dateparse(v):
                 },
             )
         )
-        if not validation_mode is ValidationMode.NONE:
+        if validation_mode is not None:
```

Comment:
It is enough to check for a boolean value here, isn't it? We don't have to _exactly_ require `None` (ditto for equivalent change in `ocm` module).

Reply:
technically, you are correct. However, I wanted to be slightly stricter here to catch programming-errors.

by explictly checking for `None` + otherwise "casting" to ValidationMode, passing-in values, such as `False`, an empty string, or 0 (or actually any value that will evaluate to `False`) would silently shortcut validation. This made me go for this more specific check. WDYT?